README.md

```diff
@@ -214,5 +214,3 @@ a List of Strings as input(s) / produces a List of String as output(s). For thes
 Triton allows users to pass String input(s)/recieve String output(s) using the String
 datatype. As a limitation of using List instead of Tensor for String I/O, only for
 1-dimensional input(s)/output(s) are supported for I/O of String type.
-Batching is not allowed for PyTorch models with String I/O. For these models,
```

Comment:
We still cannot allow 2D string I/O in the pytorch framework. Modify the docs to highlight the same

Reply:
I think this should be addressed already in the previous line where we mention the framework only supports 1D string I/O?